src/property.rs

```diff
@@ -174,21 +174,30 @@ impl<S: ContextTrait> Input<S> for Markdown {
         name: &str,
         name_human: &str,
         _required: bool,
-        _ctx: &FormRenderContext<'_, S>,
+        ctx: &FormRenderContext<'_, S>,
         _i18n: &FluentLanguageLoader,
     ) -> Markup {
         let id = Uuid::new_v4();
+        let editor_construction = ctx.ctx.editor().map(|config| {
+            format!(
+                "new EasyMDE({{ element: this, imageMaxSize: {max_size}, uploadImage: {upload}, imageUploadEndpoint: '/upload', imagePathAbsolute: true, imageAccept: '{file_types}' }})",
+                max_size = config.upload_max_size,
+                upload = config.enable_uploads,
+                file_types = config.allowed_file_types.join(", "))
```

Comment:
Should we escape `'` and `\n`? In general, are there good ways (libraries?) to do this?

Reply:
In this case, the values will almost surely only come from the developer, so it doesn't seem to necessary. But maybe it would still be a good idea. Potentially we can expect values to only consist of `[a-zA-Z/]`, which we could check...